website/ocular-config.js

```diff
@@ -19,11 +19,15 @@ const ALIASES = {
   '@nebula.gl/overlays/dist': OVERLAYS_SRC_DIR,
   '@nebula.gl/overlays': OVERLAYS_SRC_DIR,
 
+  'deck.gl': resolve(LIB_DIR, './node_modules/deck.gl'),
```

Comment:
Recommend that you stop using the top-level component completely in favor of the scoped packages. Just a quick update of code and examples.